### ISSUE TITLE
Error after beginning step 2

### ISSUE THREAD

**Issue by mti428** [SAND-Lab/MEA-NAP#58]
Hello,

I'm trying to analyze a 16-electrode Axion MEA and can get through the first step of the pipeline just fine, but it continues to give me this error. How can I fix this?

```
Error using matlab.internal.math.interp1
Sample points must be unique.

Error in interp1 (line 188)
        VqLite = matlab.internal.math.interp1(X,V,method,method,Xqcol);
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in fLOESS (line 72)
    h = interp1(hlims(:,2),hlims(:,1),x(i));
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in getISInTh (line 62)
    n = fLOESS(n, 8/round(length(n))); % used by Sit 2018
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in burstDetect (line 182)
        ISInTh = getISInTh(Spike.T, N, Steps, plotFig);
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in firingRatesBursts (line 42)
[burstMatrix, burstTimes, burstChannels] = burstDetect(spikeMatrix, ...
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Error in MEApipeline (line 373)
        Ephys = firingRatesBursts(spikeMatrix,Params,Info);
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by Timothysit** [comment]
Hi, can you send me the spike detection file that led to this error  (either here / or email me)? Thanks!

**Comment by Timothysit** [comment]
Hi again, after some playing around with the burst detection code I think one potential issue is that the spike data you have contain too few spikes (such that the number of unique inter-spike-interval times when you combine all the channels' spike time together is still less than 10, which is necessary for network burst detection). I just wrote some code to bypass that requirement (the burst detection will report no bursts) in version 1.9.1 of the pipeline (https://github.com/SAND-Lab/MEA-NAP/releases/tag/v1.9.1). Try it out and let me know how it goes!

**Comment by Timothysit** [comment]
I am closing this now as it should be resolved in version 1.9.1 and subsequent versions, but let me know if you still have this issue and I will re-open it :)
